### PR TITLE
Update contentful code for new service interface

### PR DESCRIPTION
**Pull request by dariuszor**
@lauraluiz @schleichardt 
Could you have a look?
I've updated code to meet new requirements.
I am updating README.md file.

### CODE REVIEW

File: cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsPage.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    return content;
+                });
+    }
+
+    @SuppressWarnings("unchecked")
+    private Map<String, Object> getLocaleContentMap(String fieldName, Map<String, Object> rawFields) {
+        return (Map<String, Object>) rawFields.get(fieldName);
+    }
+
+    private class FieldIdentifier {
```
Comment (lauraluiz) [inline cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsPage.java]: It's a wild guess because I haven't checked, but I think you can use an `[Iterator](https://docs.oracle.com/javase/7/docs/api/java/util/Iterator.html)` to do most of this methods (`hasNext`, `next`, `remove`). Why did you need to implement your own class?

**Comment by lauraluiz** [comment]
Besides the comment, I think the rest is good to go 👍

File: cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsPage.java (hunk 1 above)
Comment (dariuszor) [inline cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsPage.java]: Yes, most method would be covered, but this way I've got all of them grouped in one object, which is related to (field navigation)/path.

Comment (dariuszor) [inline cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsPage.java]: @lauraluiz Now I see my comment doesn't make sense.
Anyway, I removed private class here:
https://github.com/commercetools/commercetools-sunrise-java-cms/pull/11/commits/35f894d731c4f87ca043db833369f9b79561e4a8

**Comment by dariuszor** [comment]
@lauraluiz so I will merge.
Should I assign PR to you when I am updating PR according to your comments, like I did before?

I am planning to integrate this service to my sunrise project. 
It has every functionality which are required for now.

**Comment by lauraluiz** [comment]
@dariuszor sorry, the update was fine.

Regarding to Sunrise, I am currently updating the Cms behaviour from it. You won't be able to use it until I finish. I plan on finishing the changes today, then I'll create a snapshot version you can use.

**Comment by floriansattler** [comment]
@lauraluiz did you update the CMS behaviour already? Please let us know when it's finished.

**Comment by lauraluiz** [comment]
@dariuszor the new CMS structure is being used in Sunrise already, with version `0.8.2-cms-update-SNAPSHOT`. In order to use CMS from a controller, you have to implement the interface `WithCmsPage`. It is used already in `HomeController`, if you want an example.

Regardless of that, an open issue is that this CMS service is not being released yet and the common stuff needs to be removed from one of the sides (now we have 2 `CmsService` and so on). Are you guys working on some temporary solution for that already or are you expecting us to do this part?

**Comment by floriansattler** [comment]
@lauraluiz we haven't been working on a temporary solution here yet and would expect you guys to do that. Until when could you finish it?
